packages/components/inputs/search-select-input/src/search-select-input.tsx

```diff
@@ -291,6 +295,7 @@ const SearchSelectInput = (props: TSearchSelectInputProps) => {
         loadingMessage={loadingMessage}
         noOptionsMessage={noOptionsMessage}
         isSearchable={true}
+        closeMenuOnSelect={props.closeMenuOnSelect}
```

Comment:
I think it would be great to also add the default value to the `defaultProps` attribute of the component.